data/core/get_started.md

```diff
@@ -0,0 +1,175 @@
+## story_001
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* switch{"current_api": "dialogflow"}
+  - utter_switch_dialogflow
+
+## story_002
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* switch{"current_api": "luis"}
+  - utter_switch_luis
+
+## story_003
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* switch
+  - utter_quickstart_nlu_only
+* thank
+  - utter_noworries
+
+
+## story_004
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+
+## story_005
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* switch
+  - utter_quickstart_nlu_only
```

Comment:
same here, `utter_quickstart_nlu_only` shouldn't come after the `switch` intent

Reply:
what would you suggest?